scripts/mongo/loadTriples.php

```diff
@@ -9,7 +9,15 @@
 require_once 'mongo/util/TriplesUtil.class.php';
 require_once 'classes/Timer.class.php';
```

Comment:
There might be other ones that can be removed too that haven't been picked up in the diff (as they haven't been changed)